### DISCUSSION TITLE
Could I define the new `aggr` class directly? [PyG2.2.0]

### DISCUSSION THREAD

**Discussion by StefanIsSmart** [pyg-team/pytorch_geometric#8970] [General]
There are some errors happened with the PyG over 2.2.0, just as: 
https://github.com/pyg-team/pytorch_geometric/discussions/8959
https://github.com/pyg-team/pytorch_geometric/issues/8968

What I want to use is the new aggr. class `DeepSetsAggregation` and `SetTransformerAggregation`

That point is why I want to update the PyG.

Could I define it directly in my script?

**Comment by rusty1s** [comment]
If you don't want to upgrade, you can consider just copy-pasting these files. No guarantees that it works out-of-the-box though.
